content/en/docs/Configuration/namespace-management.md

```diff
@@ -5,188 +5,201 @@ description: "Configuring the namespaces accessible and visible to Kiali."
 
 ## Introduction
 
-The default Kiali installation (as mentioned in the [Installation guide]({{< ref
-"/docs/installation/installation-guide" >}})) gives Kiali access to all
-namespaces available in the cluster.
+The default Kiali [installation]({{< ref "/docs/installation/installation-guide" >}}) gives Kiali access to all namespaces available in the cluster and will allow all namespaces to be visible.
 
-It is possible to restrict Kiali to a set of desired namespaces by providing a list
-of the ones you want, excluding the ones you don't want. You can also filter namespaces by using [Istio's discovery selectors](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig). You can use a combination of these options together.
+It is possible to restrict Kiali so that it can only access a specific set of namespaces by providing [discovery selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) that match those namespaces. Note that Kiali will not use [Istio's discovery selectors](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig); if Istio has been configured with its own discovery selectors, you will likely want to configure Kiali with the same list of discovery selectors.
+
+{{% alert color="info" %}}
+This documentation makes a distinction between a namespace that is _accessible_ verses _visible_. Kiali can be given access to all namespaces in the cluster, or it can be given restricted permissions so that only certain namespaces are accessible to it. Of the accessible namespaces that Kiali is permitted to see, it can make some or all of them visible to end users.
```

Comment:
```suggestion
This documentation makes a distinction between an _accessible_ and _visible_ namespace. Kiali can be given access to all namespaces in the cluster, or it can be given restricted permissions so that only certain namespaces are accessible to it. Of the accessible namespaces that Kiali is permitted to see, it can make some or all of them visible to end users.
```

I fixed the grammatical error, "verses" is like verses of the Bible.  I think you wanted "versus", but even that's not quite right.  But more than that, I don't totally understand this blurb.  It would be better if you describe exactly the difference between an _accessible_ namespace and a _visible_ namespace.  Even I don't know what you're saying here.

Reply:
Meaning, what exactly does it mean for Kiali to have access?  What level of access, etc.

Reply:
`This documentation makes a distinction between an _accessible_ and a _visible_ namespace. The Kiali Server will be given permission to see either (a) all namespaces in the cluster, or (b) only certain namespaces in the cluster. This means the Kiali Server will be able to see the accessible namespaces (i.e. it can query for and access resources within those namespaces), but be forbidden from seeing all the other namespaces and their resources. Of these accessible namespaces that Kiali is permitted to see, it can make some or all of them visible to end users such that the end users will be able to see those namespaces and their resources in the Kiali UI.`

This is important because its the distinction between "cluster_wide_access" values of "true" and "false" which the documentation will later talk about. e.g. when you have CWA=true, the Kiali Server itself can access all namespaces in the cluster (they are "accessible") but you could still set discovery selectors to limit the visibility to only some namespaces to end users. But I didn't want to get into the details here because its all described later in the docs - but I did want to point out the distinction between "accessible" and "visible" because its important later.

Why do we do this? It's the whole cluster-scoped versus namespace-scoped caches (have a cluster-scoped cache for efficiency, have namespace-scoped caches for higher security with the downside of having many more k8s informers/watchers in the server).

Reply:
OK, how about this:
This documentation makes a distinction between _accessible_ and _visible_ namespaces. The Kiali Server will be given permission to access either (a) all, or (b) a configured subset, of cluster namespaces. The Kiali Server will only be aware of, query for, and access resources within these accessible namespaces. The set of namespaces visible to an end user, via the Kiali UI, will be a subset of the accessible namespaces.  In other words, the namespaces visible to a user may be all, or just some of the namespaces accessible to the Kiali Server..

Reply:
see latest commit